src/module/api.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-unused-vars */
```

Comment:
[OPTIONAL] Since reviewer pointed out a specific line, I will keep it as optional. But you have disabled linters in all js files except comments.js. Kindly fix them. They are there for a reason: >They help you stick to a coding style, catch errors and maintain code uniformity with your co-contributors. 
Also you can make search the related error message on eslint linters web page: https://eslint.org/docs/latest/use/getting-started